.rubocop.yml

```diff
@@ -0,0 +1,11 @@
+AllCops:
+  RunRailsCops: true
+  Exclude:
+    - db/schema.rb
```

Comment:
You can remove this line since @bwilk find that `db/schema.rb` is relevant and it will be remove when #99 is merged.